Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -31,7 +36,11 @@ public async Task<ActionResult<IEnumerable<PollutedLocationGetResponse>>> GetAll
         try
         {
             var locations = await _repository.GetAllAsync();
-            return Ok(locations);
+
+            var mappedLocations = _mapper.Map<IEnumerable<PollutedLocationGetResponse>>(locations);
```

Comment:
how does this differ from the unchanged version? are there cases where it matters? it seems like the AutoMapper handles this for us, if I'm not mistaken.

Reply:
(commenting before having looked after the tests)

Reply:
Not sure what you mean here, what is "this" that AutoMapper handles?

Reply:
![image](https://user-images.githubusercontent.com/86420873/202873203-6d0c95b3-c6bc-4bdb-a7ca-fa18a862bf63.png)
previously we had the same response type as we have now, but returned Ok with unmapped locations of type `PollutedLocation`. Is the mapping really needed, as I believe automapping takes place from when we pass `PollutedLocation`s and receive `PollutedLocationResponse`s

Reply:
No, mapping does not happen here automatically. It's just that we didn't notice because both of those objects have same properties (as of now)

Reply:
If I do this:
![image](https://user-images.githubusercontent.com/29711974/202873309-79984aac-bd66-428e-aaaf-9f3399fced81.png)
Then
![image](https://user-images.githubusercontent.com/29711974/202873315-8107464d-55eb-4a34-bb03-628665c58a2b.png)

*Note the types*